allensdk/brain_observatory/extract_running_speed/README.md

```diff
@@ -0,0 +1,24 @@
+Extract running speed
```

Comment:
Are you sure we want to start documenting our code?  This is a lot of extra work for us...